### PR TITLE
feat(themes): Blog theme tweaks

### PR DESCRIPTION
**Pull request by amberleyromo**
# Description

A few small tweaks to the blog theme and blog theme starter
- tweak query for avatar
- consolidate footer content, move away from static
- update starter modeling content shadowing for the bio and theme color changes.

## Related Issues

Related to #14276

### CODE REVIEW

**Comment by amberleyromo** [comment]
re: https://github.com/gatsbyjs/gatsby/pull/15300#pullrequestreview-256697837 -- @jxnblk, what do you think?

a few other notes @jxnblk -- I changed the way the theme color overrides are done in the theme starter. I'm doing it on the `/src/gatsby-theme-blog/gatsby-theme-ui/colors.js` level, but instead doing a deep merge of the defaults and overrides. I have a theme color mod commented out, and a note, so people using the starter can opt-in to experiment with the overrides. What do you think?

**Comment by jxnblk** [comment]
> I'm a bit new to theme-ui and I wonder why not move everything to https://theme-ui.com/sx-prop? Instead of `css={css({`

There's no reason not to, other than it potentially rocking the boat right now. Both `sx={{ ... styles }}` and `css={css({ ...styles })}` should behave in the same way, and it shouldn't be a breaking change  – I can do a pass through in another PR for people to look at

File: themes/gatsby-starter-blog-theme/src/gatsby-theme-blog/gatsby-theme-ui/colors.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import merge from "lodash.merge"
+import defaultThemeColors from "gatsby-theme-blog/src/gatsby-theme-ui/colors"
+
+/*
+ * Want to change your theme colors?
+ * Try uncommenting the color overrides below
+ * to go from default purple to a blue theme
```
Comment (jxnblk) [inline themes/gatsby-starter-blog-theme/src/gatsby-theme-blog/gatsby-theme-ui/colors.js]: ❤️ Love this! I'm a fan of code comments to show how things like this work

File: themes/gatsby-theme-blog/src/components/bio.js [hunk 2]
```diff
@@ -58,8 +58,7 @@ const bioQuery = graphql`
         author
       }
     }
-    # make this request silently fail?
-    avatar: file(absolutePath: { regex: "/avatar.jpg/" }) {
+    avatar: file(absolutePath: { regex: "/avatar.(jpeg|jpg|gif|png)/" }) {
```
Comment (jxnblk) [inline themes/gatsby-theme-blog/src/components/bio.js]: Awesome!

File: themes/gatsby-starter-blog-theme/src/gatsby-theme-blog/gatsby-theme-ui/colors.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+// const blue60 = `#007acc`
+// const blue30 = `#66E0FF`
```
Comment (jxnblk) [inline themes/gatsby-starter-blog-theme/src/gatsby-theme-blog/gatsby-theme-ui/colors.js]: Should these be more natural-language names like `blue` and `lightBlue`?

**Comment by jxnblk** [comment]
I took the liberty of removing the other blog themes in this PR as well – cc @johno @ChristopherBiscardi

**Pull request merged**